File: 151001/Danko/RESTApplication/src/main/java/com/example/restapplication/dto/MessageResponseTo.java
package com.example.restapplication.dto;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class MessageResponseTo {
    private Long id;
    private Long storyId;
    private String content;
}

File: 151004/Korol/Lab4/publisher/DC_Lab1/DC_Lab1/DTO/MessageResponseTo.cs
using DC_Lab1.DTO.Interface;

namespace DC_Lab1.DTO
{
    public class MessageResponseTo(int Id, int? storyId, string? Content, Story? Story) : IResponseTo
    {
        public int Id { get; set; } = Id;

        public int? storyId { get; set; } = storyId;

        public string? Content { get; set; } = Content;
    }
}
